src/hotspot/share/gc/g1/g1ConcurrentRebuildAndScrub.cpp

```diff
@@ -73,28 +73,16 @@ class G1RebuildRSAndScrubTask : public WorkerTask {
 
     const bool _should_rebuild_remset;
 
-    size_t _marked_words;
     size_t _processed_words;
 
     const size_t ProcessingYieldLimitInWords = G1RebuildRemSetChunkSize / HeapWordSize;
 
-    void reset_marked_words() {
-      _marked_words = 0;
-    }
-
     void reset_processed_words() {
       _processed_words = 0;
     }
 
-    void assert_marked_words(HeapRegion* hr) {
-      assert((_marked_words * HeapWordSize) == hr->marked_bytes(),
```

Comment:
This use is for verification/debugging only and has been suggested to remove (the `_marked_words` member) here during review of the single bitmap change.